common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -1,24 +1,60 @@
 ---
```

Comment:
@ahsan-ul-haq, does this work affect the work you are doing in [PR 7564](https://github.com/edx/edx-platform/pull/7564)?  I know you are working on using proper fieldset and legends for checkbox and radio button questions so that we no longer have to rely on aria-labels on paragraph elements etc.  Do we need to coordinate on this work at all?